contracts/VoucherKernel.sol

```diff
@@ -1063,6 +1059,36 @@ contract VoucherKernel is IVoucherKernel, Ownable, Pausable, UsingHelpers {
         return promises[promiseKey].depositSe;
     }
 
+    /**
+     * @notice Get the holder of a supply
+     * @param _tokenIdSupply        ID of a promise which is mapped to the corresponding Promise
+     * @return                  Address of the holder
+     */
+    function getSupplyHolder(uint256 _tokenIdSupply)
+        public
+        view
+        override
+        returns (address)
+    {
+        bytes32 promiseKey = ordersPromise[_tokenIdSupply];
+        return promises[promiseKey].seller;
+    }
+
+    /**
+     * @notice Get promise data not retrieved by other accessor functions
+     * @param _promiseKey   ID of the promise
+     * @return cancel or fault period
```

Comment:
Confused by the comment on @return. 
It looks like the function returns the data of the promise (instead of the cancel or fault period)
Please reformulate